test/unit/metrics_history_test.rb

```
@@ -184,6 +184,38 @@ def test_middleware_records_success_and_yields_return
+  def test_auto_registered_on_server_chain
+    assert Wurk.configuration.server_middleware.exists?(Wurk::Metrics::History)
+  end
+
+  # An app that also registers History in an initializer must not double-count:
+  # Chain#add removes any existing entry for the klass before appending, so
+  # re-adding replaces rather than duplicates.
+  def test_re_adding_history_keeps_a_single_entry
+    chain = Wurk::Configuration.new.server_middleware
+    chain.add(Wurk::Metrics::History)
+
+    assert_equal 1, chain.entries.count { |e| e.klass == Wurk::Metrics::History }
```

RuboCop (1.87.0)

[warning] Parenthesize the param `chain.entries.count { |e| e.klass == Wurk::Metrics::History }` to make sure that the block will be associated with the `chain.entries.count` method call.

(Lint/AmbiguousBlockAssociation)